catalog/task/create-internal-request/0.6/create-internal-request.yaml

```diff
@@ -0,0 +1,182 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-internal-request
+  labels:
+    app.kubernetes.io/version: "0.6"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Creates an InternalRequest resource to call IIB service
+  params:
+    - name: pipelineRunName
+      type: string
+      description: The name of the Parent PipelineRun of this task
+    - name: request
+      type: string
+      description: request type
+    - name: params
+      type: string
+      description: Internal Request parameters
+    - name: inputDataFile
+      type: string
+      description: Optional file to read data from
+      default: ""
+    - name: updateGenericResult
+      default: "false"
+      description: Should the task update the genericResult result
+    - name: requestUpdateTimeout
+      type: string
+      default: "360"
+      description: Max seconds waiting for the status update
+  results:
+    - name: requestMessage
+      description: Internal Request message
+    - name: requestReason
+      description: Internal Request reason
+    - name: requestResultsFile
+      description: Internal Request results file
+    - name: genericResult
+      description: genericResult data to expose to the Pipeline
```

Comment:
Can we say it is like `genericResult field from the InternalRequest Status to expose to the PipelineRun` or something a little more explanatory?